plugins/alpha-debug-pod.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: alpha-debug-pod
+spec:
+  version: v0.1.0
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/verb/kubectl-debug/releases/download/v0.1.0/kubectl-debug_darwin_amd64.tar.gz
+    sha256: "c21bb8477caa78eb9c271097346aff8a0e726aefd20c20b5ef24a49c784233f3"
+    bin: ./kubectl-debug
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/verb/kubectl-debug/releases/download/v0.1.0/kubectl-debug_linux_amd64.tar.gz
+    sha256: "098fb817c3a8ad631177008462b4bbe18bf2504bc7bc5bfd33c986b95bbf644c"
+    bin: ./kubectl-debug
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/verb/kubectl-debug/releases/download/v0.1.0/kubectl-debug_windows_amd64.zip
+    sha256: "fbf3405dad362f1d6f4e766efba4878e3988bb719710fb3701bf150a5e064a2b"
+    bin: ./kubectl-debug.exe
+  shortDescription: Attach ephemeral debug container to running pod
+  homepage: https://github.com/verb/kubectl-debug
+  caveats: |
+    This plugin requires the alpha EphemeralContainers feature to be enabled in the cluster.
+    See https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ for
+    enabling features.
+  description: |
+    This plugin attaches an Ephemeral Container to a running pod for use as an interactive
```

Comment:
Could you also add a note how to find out about the plugin usage? Like `--help` flag or so.